Makefile

```diff
@@ -782,4 +795,5 @@ $(OUTPUT_DIR)/spec/base:
 test: $(OUTPUT_DIR)/spec $(OUTPUT_DIR)/.busted
 	cd $(OUTPUT_DIR) && busted -l ./luajit --exclude-tags=notest
 
-.PHONY: test
+.PHONY: test clean fetchthirdparty \
```

Comment:
Since there is no file called `clean` `fetchthirdparty`, it's unnecessary to make them phony. Only `test` is necessary because we have a `test` directory.